expo/gbe/views.py

```diff
@@ -1380,8 +1390,12 @@ def view_volunteer(request, volunteer_id):
     volunteer = get_object_or_404(Volunteer, id=volunteer_id)
     if volunteer.profile != request.user.profile:
         validate_perms(request, ('Volunteer Reviewers',), require=True)
-    volunteerform = VolunteerBidForm(instance=volunteer,
-                                     prefix='Volunteer Info')
+    volunteerform = VolunteerBidForm(
+        instance=volunteer,
+        prefix='Volunteer Info', 
+        available_windows=volunteer.conference.windows(),
+        unavailable_windows=volunteer.conference.windows()
```

Comment:
Okay, pushed a fix to this. Next time I have to touch the readonlyform, it dies.